### PR TITLE
More updates for getting in-line with the default game

### PR DESCRIPTION
**Pull request by RQWorldblender**
A new texture is provided for newly-made compressed dirt, which is based off of the texture for the now-removed tar in this mod: 
moreblocks_dirt_compressed xcf
I'm currently trying to find out how to make this mod's glass nodes look as good as the default glass do when registered in stairsplus, but I'm having a hard time pinpointing the exact code that deals with the tiles.

### CODE REVIEW

**Comment by sofar** [comment]
These all look very good, but I haven't tested them. Are there no backwards compatibility issues though? This is quite the list of changes!

@Calinou want to take a peek?

**Comment by sofar** [comment]
rebase needed.

@RQWorldblender if you rebase this, I'll make sure it gets tested and merged.

**Comment by RQWorldblender** [comment]
I did my rebase just now, but some merge conflicts mean that some changes that I made, most noticeably that the glass textures I removed will reappear. I'm also trying to find out why some things that work well with the default nodes don't work so well with the ones found in a few mods. Here are the things I noticed so far:

- The glass introduced by this mod; already mentioned in commit c0c07ef, only looks good in the current state, but I'm having a hard time trying to find out why the default glass draws well, even when connected glass is enabled, but not for these kinds of glass.
- In the moreores mod, the copper rail works fine, except that it is possible to place those rails in midair, while it is not possible to do for the default rails. I wish I could get @Calinou to look at this as well.

I'm almost ready to have my changes committed, but I prefer waiting until the glass drawing issue is solved.

**Comment by sofar** [comment]
rebase needed, again.

File: config.lua [hunk 1]
```diff
@@ -27,3 +27,12 @@ end
 
 -- Show stairs/slabs/panels/microblocks in creative inventory (true or false):
 setting("bool", "stairsplus_in_creative_inventory", false)
+
+-- Enable customizing the tool repair value
+local tool_wear_default = -0.02
```
Comment (kaeza) [inline config.lua]: I think this line can be removed as the comment below already documents the "default" value.

**Comment by RQWorldblender** [comment]
There I go; I just merged all the recent changes onto my branch, but there haven't been any of my new changes yet.

**Pull request merged**